yt_dlp/extractor/filmarchiv.py

```diff
@@ -0,0 +1,55 @@
+from .common import InfoExtractor
+from ..utils import clean_html
+from ..utils.traversal import find_elements, traverse_obj
+
+
+class FilmArchivIE(InfoExtractor):
+    IE_NAME = 'FILMARCHIV ON'
+    _VALID_URL = r'https?://(?:www\.)?filmarchiv\.at/de/filmarchiv-on/video/(?P<id>[0-9a-zA-Z_]+)'
+    _TESTS = [{
+        'url': 'https://www.filmarchiv.at/de/filmarchiv-on/video/f_0305p7xKrXUPBwoNE9x6mh',
+        'md5': '54a6596f6a84624531866008a77fa27a',
+        'info_dict': {
+            'id': 'f_0305p7xKrXUPBwoNE9x6mh',
+            'ext': 'mp4',
+            'title': 'Der Wurstelprater zur Kaiserzeit',
+            'description': 'md5:9843f92df5cc9a4975cee7aabcf6e3b2',
+            'thumbnail': r're:https://cdn.filmarchiv.at/f_0305/p7xKrXUPBwoNE9x6mh[^/]*/poster.jpg$',
+        },
+    }]
+
+    def _real_extract(self, url):
+        media_id = self._match_id(url)
+        webpage = self._download_webpage(url, media_id)
+
+        title = self._html_search_regex(
+            r'<title-div[^>]*>\s*(.+?)\s*</title-div>',
+            webpage, 'title')
```

Comment:
```suggestion
        title = self._html_search_regex(
            r'<title-div[^>]*>\s*(.+?)\s*</title-div>', webpage, 'title')
```
can be put in one line since the regex is short enough